app/commands/git/sync.rb

```diff
@@ -42,17 +42,17 @@ def track_config_modified?
 
     memoize
     def concept_exercises_config
-      config[:exercises][:concept]
+      config[:exercises][:concept].to_a
```

Comment:
This change was not strictly necessary, but I felt it might make sense as an additional hardening step.